app/controllers/users_controller.rb

```diff
@@ -40,6 +41,10 @@ def create
     end
   end
 
+  def edit
+    @user.organization_users.build unless @user.organization_users.exists?
```

Comment:
Same question. Are we using the result from `build` anywhere?

Reply:
See above